frontend/taipy-gui/src/components/Taipy/Metric.tsx

```diff
@@ -144,34 +145,40 @@ const Metric = (props: MetricProps) => {
             layout.template = template;
         }
 
-        return layout
+        return {
+            title: props.title || layout.title,
+            ...layout
+        } as Partial<Layout>
     }, [
+        props.title,
         props.template,
         props.template_Dark_,
         props.template_Light_,
         theme.palette.mode,
-        baseLayout
+        baseLayout,
     ])
 
     return (
         <Box data-testid={props.testId} className={className}>
             <Tooltip title={hover || ""}>
```

Comment:
Move the `Tooltip` around the parent `Box`

Reply:
I've moved it around the Box in the latest commit